src/next-transpile-modules.js

```diff
@@ -73,13 +78,14 @@ const withTmInitializer = (transpileModules = []) => {
         if (config.externals) {
           config.externals = config.externals.map((external) => {
             if (typeof external !== 'function') return external;
-            return (ctx, req, cb) => {
-              return includes.find((include) =>
-                req.startsWith('.') ? include.test(path.resolve(ctx, req)) : include.test(req)
```

Comment:
@martpie re: the other comment, this is where I took it from.

Reply:
ah sorry, missed it 👍